subsys/net/lib/lwm2m/lwm2m_engine.c

```diff
@@ -3585,7 +3585,7 @@ static void retransmit_request(struct k_work *work)
 		return;
 	}
 
-	LOG_DBG("Resending message: %p", msg);
+	LOG_WRN("Resending message: %p", msg);
```

Comment:
Warning sounds a bit too much, would INF be ok here instead?

Reply:
My intent was to show it by default rather than hide.  But, you're probably right, INF is more appropriate.

Reply:
> My intent was to show it by default rather than hide.

Well, if it's normal flow of protocol, why show it by default [in a real product for example]. Otherwise, this seems like a perfect case for https://github.com/zephyrproject-rtos/zephyr/pull/8769/

Reply:
@pfalcon it's not "normal".  It does add value to be seen on console if the user is watching.  If not (as in a production) environment, then the console probably won't be on or limited to ERR anyway.

Reply:
I should clarify: you should not see this under normal operating circumstances.  You might see it under stress loads, and that's a good thing.